models/endpoints.models.js

```diff
@@ -0,0 +1,9 @@
+const fs = require("fs/promises");
+
+exports.selectEndpoints = () => {
+    return fs.readFile(`${__dirname}/../endpoints.json`).then((fileContents) => {
+        const contentsToString = fileContents.toString();
+        const endpoints = JSON.parse(contentsToString);
+        return endpoints;
+    });
```

Comment:
Your model is for making queries to the database and so this wouldn't be necessary for this ticket. You could use fs to read the file in the controller 💭